qt/widgets/plotting/src/PreviewPlot.cpp

```diff
@@ -624,18 +610,19 @@ void PreviewPlot::onWorkspaceRemoved(Mantid::API::WorkspacePreDeleteNotification
 
 /**
  * Observer method called when a workspace is replaced in the ADS
- * @param nf A pointer to the notification object
+ * @param wsName The name of the workspace which has been replaced.
+ * @param ws The new workspace.
  */
-void PreviewPlot::onWorkspaceReplaced(Mantid::API::WorkspaceBeforeReplaceNotification_ptr nf) {
+void PreviewPlot::replaceHandle(const std::string &wsName, const Workspace_sptr &ws) {
+  (void)wsName;
```

Comment:
I guess is up to your personal preference, in some other places of mantid this is written with  `UNUSED_ARG(..)` macro. Would you prefer this rather than the macro?

Reply:
Yeah good question. I guess `UNUSED_ARG` is more explicit as its clearer what its doing, but then `(void)var;` is probably easier to remember as it doesn't require a named macro. Either way I think it would just be down to preference, but I'm not sure I have a preference!

Reply:
Ok. No problem.